tests/integration/test_integration.py

```
@@ -14,11 +14,12 @@ def test_invalid_parameter(self):
         file = Path(__file__).resolve().parent.parent / "assets" / "example_audio.mp3"
 
         # when
-        response = requests.post(
-            "http://localhost:50001/v1/audio/transcriptions",
-            files={"file": open(file, "rb")},
-            data={"schmutzgeier": "schmutzgeier"},
-        )
+        with open(file, "rb") as f:
+            response = requests.post(
```

Ruff (0.15.21)

[error] Probable use of `requests` call without timeout

(S113)